[[1;34mINFO[m] Scanning for projects...
[[1;33mWARNING[m] 
[[1;33mWARNING[m] Some problems were encountered while building the effective model for org.apache.hadoop:hadoop-common:jar:3.3.3
[[1;33mWARNING[m] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.apache.hbase:hbase-server:jar -> duplicate declaration of version ${hbase.version} @ org.apache.hadoop:hadoop-project:3.3.3, /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-project/pom.xml, line 1691, column 19
[[1;33mWARNING[m] 
[[1;33mWARNING[m] It is highly recommended to fix these problems because they threaten the stability of your build.
[[1;33mWARNING[m] 
[[1;33mWARNING[m] For this reason, future Maven versions might no longer support building such malformed projects.
[[1;33mWARNING[m] 
[[1;34mINFO[m] ------------------------------------------------------------------------
[[1;34mINFO[m] Detecting the operating system and CPU architecture
[[1;34mINFO[m] ------------------------------------------------------------------------
[[1;34mINFO[m] os.detected.name: linux
[[1;34mINFO[m] os.detected.arch: x86_64
[[1;34mINFO[m] os.detected.version: 5.10
[[1;34mINFO[m] os.detected.version.major: 5
[[1;34mINFO[m] os.detected.version.minor: 10
[[1;34mINFO[m] os.detected.release: ubuntu
[[1;34mINFO[m] os.detected.release.version: 20.04
[[1;34mINFO[m] os.detected.release.like.ubuntu: true
[[1;34mINFO[m] os.detected.release.like.debian: true
[[1;34mINFO[m] os.detected.classifier: linux-x86_64
[[1;34mINFO[m] 
[[1;34mINFO[m] [1m------------------< [0;36morg.apache.hadoop:hadoop-common[0;1m >-------------------[m
[[1;34mINFO[m] [1mBuilding Apache Hadoop Common 3.3.3[m
[[1;34mINFO[m] [1m--------------------------------[ jar ]---------------------------------[m
[[1;34mINFO[m] 
[[1;34mINFO[m] [1m--- [0;32mjqf-maven-plugin:2.0-SNAPSHOT:fuzz[m [1m(default-cli)[m @ [36mhadoop-common[0;1m ---[m
Set Maven-Surefire-Plugin Configuration
[[1;34mINFO[m] Using port 13116
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] Server environment:zookeeper.version=3.5.6-c11b7e26bc554b8523dc929761dd28808913f091, built on 10/08/2019 20:18 GMT
[[1;34mINFO[m] Server environment:host.name=KingsLand
[[1;34mINFO[m] Server environment:java.version=11.0.15
[[1;34mINFO[m] Server environment:java.vendor=Private Build
[[1;34mINFO[m] Server environment:java.home=/usr/lib/jvm/java-11-openjdk-amd64
[[1;34mINFO[m] Server environment:java.class.path=/usr/share/maven/boot/plexus-classworlds-2.x.jar
[[1;34mINFO[m] Server environment:java.library.path=/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib
[[1;34mINFO[m] Server environment:java.io.tmpdir=/tmp
[[1;34mINFO[m] Server environment:java.compiler=<NA>
[[1;34mINFO[m] Server environment:os.name=Linux
[[1;34mINFO[m] Server environment:os.arch=amd64
[[1;34mINFO[m] Server environment:os.version=5.10.0-1050-oem
[[1;34mINFO[m] Server environment:user.name=shuai
[[1;34mINFO[m] Server environment:user.home=/home/shuai
[[1;34mINFO[m] Server environment:user.dir=/home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common
[[1;34mINFO[m] Server environment:os.memory.free=452MB
[[1;34mINFO[m] Server environment:os.memory.max=7960MB
[[1;34mINFO[m] Server environment:os.memory.total=502MB
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test11808736327006637817.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test11808736327006637817.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:13116
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test11808736327006637817.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test11808736327006637817.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 13116
[[1;34mINFO[m] The list of known four letter word commands is : [{1936881266=srvr, 1937006964=stat, 2003003491=wchc, 1685417328=dump, 1668445044=crst, 1936880500=srst, 1701738089=envi, 1668247142=conf, -720899=telnet close, 2003003507=wchs, 2003003504=wchp, 1684632179=dirs, 1668247155=cons, 1835955314=mntr, 1769173615=isro, 1920298859=ruok, 1735683435=gtmk, 1937010027=stmk}]
[[1;34mINFO[m] The list of enabled four letter word commands is : [[wchs, stat, wchp, dirs, stmk, conf, ruok, mntr, srvr, wchc, envi, srst, isro, dump, gtmk, telnet close, crst, cons]]
[[1;34mINFO[m] Processing stat command from /127.0.0.1:43066
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;33mWARNING[m] Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[[1;34mINFO[m] Client environment:zookeeper.version=3.5.6-c11b7e26bc554b8523dc929761dd28808913f091, built on 10/08/2019 20:18 GMT
[[1;34mINFO[m] Client environment:host.name=KingsLand
[[1;34mINFO[m] Client environment:java.version=11.0.15
[[1;34mINFO[m] Client environment:java.vendor=Private Build
[[1;34mINFO[m] Client environment:java.home=/usr/lib/jvm/java-11-openjdk-amd64
[[1;34mINFO[m] Client environment:java.class.path=/usr/share/maven/boot/plexus-classworlds-2.x.jar
[[1;34mINFO[m] Client environment:java.library.path=/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib
[[1;34mINFO[m] Client environment:java.io.tmpdir=/tmp
[[1;34mINFO[m] Client environment:java.compiler=<NA>
[[1;34mINFO[m] Client environment:os.name=Linux
[[1;34mINFO[m] Client environment:os.arch=amd64
[[1;34mINFO[m] Client environment:os.version=5.10.0-1050-oem
[[1;34mINFO[m] Client environment:user.name=shuai
[[1;34mINFO[m] Client environment:user.home=/home/shuai
[[1;34mINFO[m] Client environment:user.dir=/home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common
[[1;34mINFO[m] Client environment:os.memory.free=200MB
[[1;34mINFO[m] Client environment:os.memory.max=7960MB
[[1;34mINFO[m] Client environment:os.memory.total=502MB
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:13116 sessionTimeout=10000 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@6b8d5a56
[[1;34mINFO[m] Setting -D jdk.tls.rejectClientInitiatedRenegotiation=true to disable client-initiated TLS renegotiation
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:13116. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:43114, server: localhost/127.0.0.1:13116
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:13116, sessionid = 0x1020c660a000000, negotiated timeout = 10000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:43114
[[1;34mINFO[m] auth success /127.0.0.1:43114
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c660a000000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c660a000000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c660a000000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:13116 sessionTimeout=10000 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@14342e1c
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:13116. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:43116, server: localhost/127.0.0.1:13116
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:13116, sessionid = 0x1020c660a000001, negotiated timeout = 10000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:43116
[[1;34mINFO[m] auth success /127.0.0.1:43116
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c660a000001 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c660a000001
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] EventThread shut down for session: 0x1020c660a000001
[[1;34mINFO[m] STOPPING server
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 13116
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[JQF] After pre round flag = false
[JQF] After preRound mapping size = 32
[[1;34mINFO[m] Using port 41257
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test2344817637033927702.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test2344817637033927702.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:41257
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test2344817637033927702.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test2344817637033927702.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 41257
[[1;34mINFO[m] Processing stat command from /127.0.0.1:37844
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:41257 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@56bb387c
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:41257. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:37850, server: localhost/127.0.0.1:41257
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:41257, sessionid = 0x1020c6613c90000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:37850
[[1;34mINFO[m] auth success /127.0.0.1:37850
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6613c90000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6613c90000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6613c90000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:41257 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@5a720957
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:41257. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:37854, server: localhost/127.0.0.1:41257
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:41257, sessionid = 0x1020c6613c90001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:37854
[[1;34mINFO[m] auth success /127.0.0.1:37854
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6613c90001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6613c90001
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6613c90001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 41257
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 46238
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test9207326947157569324.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test9207326947157569324.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:46238
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test9207326947157569324.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test9207326947157569324.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 46238
[[1;34mINFO[m] Processing stat command from /127.0.0.1:44762
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:46238 sessionTimeout=1772201771 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@53c520da
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:46238. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44766, server: localhost/127.0.0.1:46238
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:46238, sessionid = 0x1020c6615fa0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44766
[[1;34mINFO[m] auth success /127.0.0.1:44766
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6615fa0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6615fa0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6615fa0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:46238 sessionTimeout=1772201771 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@2841a16c
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:46238. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44768, server: localhost/127.0.0.1:46238
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:46238, sessionid = 0x1020c6615fa0001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44768
[[1;34mINFO[m] auth success /127.0.0.1:44768
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6615fa0001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6615fa0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6615fa0001
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 46238
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 5111
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test2942310471276218033.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test2942310471276218033.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:5111
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test2942310471276218033.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test2942310471276218033.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 5111
[[1;34mINFO[m] Processing stat command from /127.0.0.1:51774
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:5111 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@7be620d8
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:5111. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:51780, server: localhost/127.0.0.1:5111
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:5111, sessionid = 0x1020c6617e90000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:51780
[[1;34mINFO[m] auth success /127.0.0.1:51780
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6617e90000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6617e90000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6617e90000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:5111 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@300ded25
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:5111. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:51782, server: localhost/127.0.0.1:5111
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:5111, sessionid = 0x1020c6617e90001, negotiated timeout = 60000
[[1;34mINFO[m] got auth packet /127.0.0.1:51782
[[1;34mINFO[m] auth success /127.0.0.1:51782
[[1;34mINFO[m] Session connected.
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6617e90001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6617e90001
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6617e90001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 5111
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 45347
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test4548647233648704841.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test4548647233648704841.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:45347
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test4548647233648704841.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test4548647233648704841.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 45347
[[1;34mINFO[m] Processing stat command from /127.0.0.1:44746
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:45347 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@65af80fe
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:45347. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44748, server: localhost/127.0.0.1:45347
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:45347, sessionid = 0x1020c6619b50000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44748
[[1;34mINFO[m] auth success /127.0.0.1:44748
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6619b50000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6619b50000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6619b50000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:45347 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@72d150fb
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:45347. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44750, server: localhost/127.0.0.1:45347
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:45347, sessionid = 0x1020c6619b50001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44750
[[1;34mINFO[m] auth success /127.0.0.1:44750
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6619b50001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6619b50001
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6619b50001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 45347
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 30617
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1747907450490845802.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1747907450490845802.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:30617
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1747907450490845802.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1747907450490845802.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 30617
[[1;34mINFO[m] Processing stat command from /127.0.0.1:40018
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:30617 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@48de9b83
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:30617. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:40020, server: localhost/127.0.0.1:30617
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:30617, sessionid = 0x1020c661b7d0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:40020
[[1;34mINFO[m] auth success /127.0.0.1:40020
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] Session: 0x1020c661b7d0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c661b7d0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c661b7d0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:30617 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@5a65d9f6
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:30617. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:40026, server: localhost/127.0.0.1:30617
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:30617, sessionid = 0x1020c661b7d0001, negotiated timeout = 60000
[[1;34mINFO[m] got auth packet /127.0.0.1:40026
[[1;34mINFO[m] auth success /127.0.0.1:40026
[[1;34mINFO[m] Session connected.
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c661b7d0001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c661b7d0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] EventThread shut down for session: 0x1020c661b7d0001
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 30617
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 11756
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5083275316426225864.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5083275316426225864.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:11756
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5083275316426225864.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5083275316426225864.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 11756
[[1;34mINFO[m] Processing stat command from /127.0.0.1:35150
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:11756 sessionTimeout=1628337925 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@5a165df4
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:11756. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:35152, server: localhost/127.0.0.1:11756
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:11756, sessionid = 0x1020c661d410000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:35152
[[1;34mINFO[m] auth success /127.0.0.1:35152
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c661d410000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c661d410000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c661d410000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:11756 sessionTimeout=1628337925 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@767b9e9c
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:11756. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:35154, server: localhost/127.0.0.1:11756
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:11756, sessionid = 0x1020c661d410001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:35154
[[1;34mINFO[m] auth success /127.0.0.1:35154
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c661d410001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c661d410001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] EventThread shut down for session: 0x1020c661d410001
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 11756
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 4665
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test13084793849666080459.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test13084793849666080459.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:4665
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test13084793849666080459.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test13084793849666080459.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 4665
[[1;34mINFO[m] Processing stat command from /127.0.0.1:44256
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:4665 sessionTimeout=1762496668 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@1b59f5fa
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:4665. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44264, server: localhost/127.0.0.1:4665
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:4665, sessionid = 0x1020c661f290000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44264
[[1;34mINFO[m] auth success /127.0.0.1:44264
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c661f290000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c661f290000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c661f290000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:4665 sessionTimeout=1762496668 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@4ac6891
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:4665. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44268, server: localhost/127.0.0.1:4665
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:4665, sessionid = 0x1020c661f290001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44268
[[1;34mINFO[m] auth success /127.0.0.1:44268
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c661f290001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c661f290001
[[1;34mINFO[m] EventThread shut down for session: 0x1020c661f290001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 4665
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 24622
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test11652876560639188435.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test11652876560639188435.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:24622
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test11652876560639188435.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test11652876560639188435.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 24622
[[1;34mINFO[m] Processing stat command from /127.0.0.1:47500
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:24622 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@319847fe
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:24622. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:47502, server: localhost/127.0.0.1:24622
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:24622, sessionid = 0x1020c6620ce0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:47502
[[1;34mINFO[m] auth success /127.0.0.1:47502
[[1;34mINFO[m] Successfully created /uiuc/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6620ce0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6620ce0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6620ce0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:24622 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@78ba6a49
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:24622. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:47504, server: localhost/127.0.0.1:24622
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:24622, sessionid = 0x1020c6620ce0001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:47504
[[1;34mINFO[m] auth success /127.0.0.1:47504
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6620ce0001 closed
[[1;34mINFO[m] tearDown starting
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6620ce0001
[[1;34mINFO[m] STOPPING server
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6620ce0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 24622
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 26903
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5770804690229759432.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5770804690229759432.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:26903
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5770804690229759432.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5770804690229759432.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 26903
[[1;34mINFO[m] Processing stat command from /127.0.0.1:59074
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:26903 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@4af66fbe
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:26903. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:59076, server: localhost/127.0.0.1:26903
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:26903, sessionid = 0x1020c6622ae0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:59076
[[1;34mINFO[m] auth success /127.0.0.1:59076
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6622ae0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6622ae0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6622ae0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:26903 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@72193b1a
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:26903. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:59080, server: localhost/127.0.0.1:26903
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:26903, sessionid = 0x1020c6622ae0001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:59080
[[1;34mINFO[m] auth success /127.0.0.1:59080
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6622ae0001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6622ae0001
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6622ae0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 26903
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 55895
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1449902589901339084.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1449902589901339084.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:55895
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1449902589901339084.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1449902589901339084.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 55895
[[1;34mINFO[m] Processing stat command from /127.0.0.1:52094
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:55895 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@6f13a13a
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:55895. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:52096, server: localhost/127.0.0.1:55895
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:55895, sessionid = 0x1020c66246a0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:52096
[[1;34mINFO[m] auth success /127.0.0.1:52096
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c66246a0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c66246a0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c66246a0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:55895 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@4cd32cc1
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:55895. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:52098, server: localhost/127.0.0.1:55895
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:55895, sessionid = 0x1020c66246a0001, negotiated timeout = 60000
[[1;34mINFO[m] got auth packet /127.0.0.1:52098
[[1;34mINFO[m] auth success /127.0.0.1:52098
[[1;34mINFO[m] Session connected.
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c66246a0001 closed
[[1;34mINFO[m] tearDown starting
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c66246a0001
[[1;34mINFO[m] STOPPING server
[[1;34mINFO[m] EventThread shut down for session: 0x1020c66246a0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 55895
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 8204
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test727368745944446162.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test727368745944446162.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:8204
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test727368745944446162.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test727368745944446162.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 8204
[[1;34mINFO[m] Processing stat command from /127.0.0.1:44038
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:8204 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@26c7a941
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:8204. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44040, server: localhost/127.0.0.1:8204
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:8204, sessionid = 0x1020c6626510000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44040
[[1;34mINFO[m] auth success /127.0.0.1:44040
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6626510000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6626510000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6626510000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:8204 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@5ab3ae6e
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:8204. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44044, server: localhost/127.0.0.1:8204
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:8204, sessionid = 0x1020c6626510001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44044
[[1;34mINFO[m] auth success /127.0.0.1:44044
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6626510001 closed
[[1;34mINFO[m] tearDown starting
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6626510001
[[1;34mINFO[m] STOPPING server
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6626510001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 8204
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] Using port 25737
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5424617003943697514.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5424617003943697514.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:25737
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5424617003943697514.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test5424617003943697514.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 25737
[[1;34mINFO[m] Processing stat command from /127.0.0.1:34086
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:25737 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@65e8ea3f
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:25737. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:34088, server: localhost/127.0.0.1:25737
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:25737, sessionid = 0x1020c66281d0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:34088
[[1;34mINFO[m] auth success /127.0.0.1:34088
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c66281d0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c66281d0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c66281d0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:25737 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@53dda23a
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:25737. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:34090, server: localhost/127.0.0.1:25737
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:25737, sessionid = 0x1020c66281d0001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:34090
[[1;34mINFO[m] auth success /127.0.0.1:34090
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c66281d0001 closed
[[1;34mINFO[m] tearDown starting
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c66281d0001
[[1;34mINFO[m] STOPPING server
[[1;34mINFO[m] EventThread shut down for session: 0x1020c66281d0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 25737
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 48578
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test15006827460110778827.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test15006827460110778827.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:48578
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test15006827460110778827.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test15006827460110778827.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 48578
[[1;34mINFO[m] Processing stat command from /127.0.0.1:44058
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:48578 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@648e136d
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:48578. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44060, server: localhost/127.0.0.1:48578
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:48578, sessionid = 0x1020c6629da0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44060
[[1;34mINFO[m] auth success /127.0.0.1:44060
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6629da0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6629da0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6629da0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:48578 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@4892f57
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:48578. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44062, server: localhost/127.0.0.1:48578
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:48578, sessionid = 0x1020c6629da0001, negotiated timeout = 60000
[[1;34mINFO[m] got auth packet /127.0.0.1:44062
[[1;34mINFO[m] auth success /127.0.0.1:44062
[[1;34mINFO[m] Session connected.
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6629da0001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6629da0001
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6629da0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 48578
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 8463
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test13616541092130156354.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test13616541092130156354.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:8463
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test13616541092130156354.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test13616541092130156354.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 8463
[[1;34mINFO[m] Processing stat command from /127.0.0.1:32892
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:8463 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@55ee78b9
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:8463. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:32896, server: localhost/127.0.0.1:8463
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:8463, sessionid = 0x1020c662b9e0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:32896
[[1;34mINFO[m] auth success /127.0.0.1:32896
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c662b9e0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c662b9e0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c662b9e0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:8463 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@1074f0e9
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:8463. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:32900, server: localhost/127.0.0.1:8463
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:8463, sessionid = 0x1020c662b9e0001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:32900
[[1;34mINFO[m] auth success /127.0.0.1:32900
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c662b9e0001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c662b9e0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] EventThread shut down for session: 0x1020c662b9e0001
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 8463
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 40736
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test9969366793042099300.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test9969366793042099300.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:40736
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test9969366793042099300.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test9969366793042099300.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 40736
[[1;34mINFO[m] Processing stat command from /127.0.0.1:42396
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:40736 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@78580bed
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:40736. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:42398, server: localhost/127.0.0.1:40736
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:40736, sessionid = 0x1020c662d640000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:42398
[[1;34mINFO[m] auth success /127.0.0.1:42398
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c662d640000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c662d640000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c662d640000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:40736 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@5f53a63c
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:40736. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:42402, server: localhost/127.0.0.1:40736
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:40736, sessionid = 0x1020c662d640001, negotiated timeout = 60000
[[1;34mINFO[m] got auth packet /127.0.0.1:42402
[[1;34mINFO[m] auth success /127.0.0.1:42402
[[1;34mINFO[m] Session connected.
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 ([1mNative Method[m)
    [1mat[m jdk.internal.reflect.NativeMethodAccessorImpl.invoke ([1mNativeMethodAccessorImpl.java:62[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c662d640001 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c662d640001
[[1;34mINFO[m] EventThread shut down for session: 0x1020c662d640001
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 40736
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 9328
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test10918207088735419777.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test10918207088735419777.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:9328
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test10918207088735419777.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test10918207088735419777.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 9328
[[1;34mINFO[m] Processing stat command from /127.0.0.1:38722
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:9328 sessionTimeout=1309505285 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@6b1fcc8f
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:9328. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:38726, server: localhost/127.0.0.1:9328
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:9328, sessionid = 0x1020c662f370000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:38726
[[1;34mINFO[m] auth success /127.0.0.1:38726
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c662f370000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c662f370000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c662f370000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:9328 sessionTimeout=1309505285 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@683f86f4
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:9328. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:38732, server: localhost/127.0.0.1:9328
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:9328, sessionid = 0x1020c662f370001, negotiated timeout = 60000
[[1;34mINFO[m] got auth packet /127.0.0.1:38732
[[1;34mINFO[m] auth success /127.0.0.1:38732
[[1;34mINFO[m] Session connected.
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.GeneratedMethodAccessor41.invoke ([1mUnknown Source[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c662f370001 closed
[[1;34mINFO[m] tearDown starting
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c662f370001
[[1;34mINFO[m] STOPPING server
[[1;34mINFO[m] EventThread shut down for session: 0x1020c662f370001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 9328
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 2367
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1771639937800043757.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1771639937800043757.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:2367
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1771639937800043757.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test1771639937800043757.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 2367
[[1;34mINFO[m] Processing stat command from /127.0.0.1:48402
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:2367 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@6f2806e0
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:2367. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:48404, server: localhost/127.0.0.1:2367
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:2367, sessionid = 0x1020c6630de0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:48404
[[1;34mINFO[m] auth success /127.0.0.1:48404
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c6630de0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6630de0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6630de0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:2367 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@32fb2d2e
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:2367. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:48408, server: localhost/127.0.0.1:2367
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:2367, sessionid = 0x1020c6630de0001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:48408
[[1;34mINFO[m] auth success /127.0.0.1:48408
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.GeneratedMethodAccessor41.invoke ([1mUnknown Source[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6630de0001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6630de0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6630de0001
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 2367
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 6845
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test7209388474893913005.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test7209388474893913005.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:6845
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test7209388474893913005.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test7209388474893913005.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 6845
[[1;34mINFO[m] Processing stat command from /127.0.0.1:59428
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:6845 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@28434852
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:6845. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:59432, server: localhost/127.0.0.1:6845
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:6845, sessionid = 0x1020c6632910000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:59432
[[1;34mINFO[m] auth success /127.0.0.1:59432
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] Session: 0x1020c6632910000 closed
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6632910000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6632910000
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] SessionTrackerImpl exited loop!
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:6845 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@16678eb6
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:6845. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:59434, server: localhost/127.0.0.1:6845
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:6845, sessionid = 0x1020c6632910001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:59434
[[1;34mINFO[m] auth success /127.0.0.1:59434
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.GeneratedMethodAccessor41.invoke ([1mUnknown Source[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c6632910001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c6632910001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] EventThread shut down for session: 0x1020c6632910001
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 6845
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 1209
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test4061190528109213578.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test4061190528109213578.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:1209
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test4061190528109213578.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test4061190528109213578.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 1209
[[1;34mINFO[m] Processing stat command from /127.0.0.1:44512
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:1209 sessionTimeout=17004293 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@44248a21
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:1209. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44516, server: localhost/127.0.0.1:1209
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:1209, sessionid = 0x1020c66346c0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44516
[[1;34mINFO[m] auth success /127.0.0.1:44516
[[1;34mINFO[m] Successfully created /hadoop-ha/dummy-cluster in ZK.
[[1;34mINFO[m] Session: 0x1020c66346c0000 closed
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c66346c0000
[[1;34mINFO[m] EventThread shut down for session: 0x1020c66346c0000
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:1209 sessionTimeout=17004293 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@1c619623
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:1209. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44518, server: localhost/127.0.0.1:1209
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:1209, sessionid = 0x1020c66346c0001, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;34mINFO[m] got auth packet /127.0.0.1:44518
[[1;34mINFO[m] auth success /127.0.0.1:44518
[[1;31mERROR[m] Fencing is not configured for DummyHAService #1.
You must configure a fencing method before using automatic failover.
[1;31morg.apache.hadoop.ha.BadFencingConfigurationException[m: [1;31mno fencing[m
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:241[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:231[m)
    [1mat[m jdk.internal.reflect.GeneratedMethodAccessor41.invoke ([1mUnknown Source[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c66346c0001 closed
[[1;34mINFO[m] tearDown starting
[[1;34mINFO[m] STOPPING server
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c66346c0001
[[1;34mINFO[m] EventThread shut down for session: 0x1020c66346c0001
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 1209
[[1;34mINFO[m] SUCCEEDED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] Using port 24876
[[1;34mINFO[m] STARTING testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] STARTING server
[[1;34mINFO[m] zookeeper.snapshot.trust.empty : false
[[1;34mINFO[m] zookeeper.snapshotSizeFactor = 0.33
[[1;34mINFO[m] minSessionTimeout set to 6000
[[1;34mINFO[m] maxSessionTimeout set to 60000
[[1;34mINFO[m] Created server with tickTime 3000 minSessionTimeout 6000 maxSessionTimeout 60000 datadir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test14693913373639767116.junit.dir/version-2 snapdir /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test14693913373639767116.junit.dir/version-2
[[1;34mINFO[m] Using org.apache.zookeeper.server.NIOServerCnxnFactory as server connection factory
[[1;34mINFO[m] Configuring NIO connection handler with 10s sessionless connection timeout, 2 selector thread(s), 32 worker threads, and 64 kB direct buffers.
[[1;34mINFO[m] binding to port 0.0.0.0/0.0.0.0:24876
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test14693913373639767116.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] Snapshotting: 0x0 to /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/target/test/data/test14693913373639767116.junit.dir/version-2/snapshot.0
[[1;34mINFO[m] connecting to 127.0.0.1 24876
[[1;34mINFO[m] Processing stat command from /127.0.0.1:44948
[[1;34mINFO[m] Stat command output
[[1;34mINFO[m] Client test setup finished
[[1;34mINFO[m] Initiating client connection, connectString=127.0.0.1:24876 sessionTimeout=1762490117 watcher=org.apache.hadoop.ha.ActiveStandbyElector$WatcherWithClientRef@3cae2612
[[1;34mINFO[m] jute.maxbuffer value is 4194304 Bytes
[[1;34mINFO[m] zookeeper.request.timeout value is 0. feature enabled=
[[1;34mINFO[m] Opening socket connection to server localhost/127.0.0.1:24876. Will not attempt to authenticate using SASL (unknown error)
[[1;34mINFO[m] Socket connection established, initiating session, client: /127.0.0.1:44950, server: localhost/127.0.0.1:24876
[[1;34mINFO[m] Creating new log file: log.1
[[1;34mINFO[m] Session establishment complete on server localhost/127.0.0.1:24876, sessionid = 0x1020c66363e0000, negotiated timeout = 60000
[[1;34mINFO[m] Session connected.
[[1;31mERROR[m] The failover controller encounters runtime error
[1;31mjava.io.IOException[m: [1;31mCouldn't create /hadoop-ha/dummy-cluster[m
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.ensureParentZNode ([1mActiveStandbyElector.java:361[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.formatZK ([1mZKFailoverController.java:292[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:222[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:228[m)
    [1mat[m jdk.internal.reflect.GeneratedMethodAccessor41.invoke ([1mUnknown Source[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[1mCaused by[m: org.apache.zookeeper.KeeperException$NoAuthException: [1;31mKeeperErrorCode = NoAuth for /hadoop-ha/dummy-cluster[m
    [1mat[m org.apache.zookeeper.KeeperException.create ([1mKeeperException.java:120[m)
    [1mat[m org.apache.zookeeper.KeeperException.create ([1mKeeperException.java:54[m)
    [1mat[m org.apache.zookeeper.ZooKeeper.create ([1mZooKeeper.java:1538[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector$3.run ([1mActiveStandbyElector.java:1041[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector$3.run ([1mActiveStandbyElector.java:1038[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.zkDoWithRetries ([1mActiveStandbyElector.java:1103[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.zkDoWithRetries ([1mActiveStandbyElector.java:1095[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.createWithRetries ([1mActiveStandbyElector.java:1038[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.ensureParentZNode ([1mActiveStandbyElector.java:350[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.formatZK ([1mZKFailoverController.java:292[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:222[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:228[m)
    [1mat[m jdk.internal.reflect.GeneratedMethodAccessor41.invoke ([1mUnknown Source[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] Session: 0x1020c66363e0000 closed
[[1;34mINFO[m] tearDown starting
[[1;33mWARNING[m] Ignoring stale result from old client with sessionId 0x1020c66363e0000
[[1;34mINFO[m] STOPPING server
[[1;34mINFO[m] EventThread shut down for session: 0x1020c66363e0000
[[1;34mINFO[m] ConnnectionExpirerThread interrupted
[[1;34mINFO[m] accept thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] selector thread exitted run method
[[1;34mINFO[m] shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] Shutting down
[[1;34mINFO[m] PrepRequestProcessor exited loop!
[[1;34mINFO[m] SyncRequestProcessor exited!
[[1;34mINFO[m] shutdown of request processor complete
[[1;34mINFO[m] connecting to 127.0.0.1 24876
[[1;34mINFO[m] FAILED testFencingMustBeConfiguredFuzz
[1;31mjava.io.IOException[m: [1;31mCouldn't create /hadoop-ha/dummy-cluster[m
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.ensureParentZNode ([1mActiveStandbyElector.java:361[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.formatZK ([1mZKFailoverController.java:292[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:222[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:228[m)
    [1mat[m jdk.internal.reflect.GeneratedMethodAccessor41.invoke ([1mUnknown Source[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[1mCaused by[m: org.apache.zookeeper.KeeperException$NoAuthException: [1;31mKeeperErrorCode = NoAuth for /hadoop-ha/dummy-cluster[m
    [1mat[m org.apache.zookeeper.KeeperException.create ([1mKeeperException.java:120[m)
    [1mat[m org.apache.zookeeper.KeeperException.create ([1mKeeperException.java:54[m)
    [1mat[m org.apache.zookeeper.ZooKeeper.create ([1mZooKeeper.java:1538[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector$3.run ([1mActiveStandbyElector.java:1041[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector$3.run ([1mActiveStandbyElector.java:1038[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.zkDoWithRetries ([1mActiveStandbyElector.java:1103[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.zkDoWithRetries ([1mActiveStandbyElector.java:1095[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.createWithRetries ([1mActiveStandbyElector.java:1038[m)
    [1mat[m org.apache.hadoop.ha.ActiveStandbyElector.ensureParentZNode ([1mActiveStandbyElector.java:350[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.formatZK ([1mZKFailoverController.java:292[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.doRun ([1mZKFailoverController.java:222[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.access$000 ([1mZKFailoverController.java:63[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:181[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController$1.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.security.SecurityUtil.doAsLoginUserOrFatal ([1mSecurityUtil.java:503[m)
    [1mat[m org.apache.hadoop.ha.ZKFailoverController.run ([1mZKFailoverController.java:177[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.runFC ([1mTestZKFailoverController.java:717[m)
    [1mat[m org.apache.hadoop.ha.TestZKFailoverController.testFencingMustBeConfiguredFuzz ([1mTestZKFailoverController.java:228[m)
    [1mat[m jdk.internal.reflect.GeneratedMethodAccessor41.invoke ([1mUnknown Source[m)
    [1mat[m jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke ([1mDelegatingMethodAccessorImpl.java:43[m)
    [1mat[m java.lang.reflect.Method.invoke ([1mMethod.java:566[m)
    [1mat[m org.junit.runners.model.FrameworkMethod$1.runReflectiveCall ([1mFrameworkMethod.java:59[m)
    [1mat[m org.junit.internal.runners.model.ReflectiveCallable.run ([1mReflectiveCallable.java:12[m)
    [1mat[m org.junit.runners.model.FrameworkMethod.invokeExplosively ([1mFrameworkMethod.java:56[m)
    [1mat[m edu.berkeley.cs.jqf.fuzz.junit.TrialRunner$1.evaluate ([1mTrialRunner.java:59[m)
    [1mat[m org.junit.internal.runners.statements.RunBefores.evaluate ([1mRunBefores.java:26[m)
    [1mat[m org.junit.internal.runners.statements.RunAfters.evaluate ([1mRunAfters.java:27[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:288[m)
    [1mat[m org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call ([1mFailOnTimeout.java:282[m)
    [1mat[m java.util.concurrent.FutureTask.run ([1mFutureTask.java:264[m)
    [1mat[m java.lang.Thread.run ([1mThread.java:829[m)
[[1;34mINFO[m] FINISHED testFencingMustBeConfiguredFuzz
[[1;34mINFO[m] [1m------------------------------------------------------------------------[m
[[1;34mINFO[m] [1;31mBUILD FAILURE[m
[[1;34mINFO[m] [1m------------------------------------------------------------------------[m
[[1;34mINFO[m] Total time:  13.913 s
[[1;34mINFO[m] Finished at: 2022-10-16T16:53:12-05:00
[[1;34mINFO[m] [1m------------------------------------------------------------------------[m
[[1;31mERROR[m] Failed to execute goal [32medu.berkeley.cs.jqf:jqf-maven-plugin:2.0-SNAPSHOT:fuzz[m [1m(default-cli)[m on project [36mhadoop-common[m: [1;31mFuzzing resulted in the test failing on 1 input(s). Possible bugs found. Use mvn jqf:repro to reproduce failing test cases from /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/fuzz_output_round2/fuzz-results/org.apache.hadoop.ha.TestZKFailoverController/testFencingMustBeConfiguredFuzz/failures. Sample exception included with this message.[m: Couldn't create /hadoop-ha/dummy-cluster: KeeperErrorCode = NoAuth for /hadoop-ha/dummy-cluster -> [1m[Help 1][m
[[1;31mERROR[m] 
[[1;31mERROR[m] To see the full stack trace of the errors, re-run Maven with the [1m-e[m switch.
[[1;31mERROR[m] Re-run Maven using the [1m-X[m switch to enable full debug logging.
[[1;31mERROR[m] 
[[1;31mERROR[m] For more information about the errors and possible solutions, please read the following articles:
[[1;31mERROR[m] [1m[Help 1][m http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
